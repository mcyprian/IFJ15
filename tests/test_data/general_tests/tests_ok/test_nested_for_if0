int main()
{
    for(int i = 0 ; i < 5; i = i + 1)
    {
            if( (i+2) / 2 == 1)
            {
                cout << i << "\n";
            }
            else
            {
               for(int j = 0 ; j < 3; j = j + 1)
                {
                
                    cout << j;    
                 }
                 cout << "\n";
            }

    }

    return 42;
}
