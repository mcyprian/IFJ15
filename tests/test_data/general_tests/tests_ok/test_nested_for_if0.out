0
1
012
012
012
